care/facility/api/viewsets/patient.py

```diff
@@ -410,7 +410,12 @@ def filter_queryset(self, queryset: QuerySet) -> QuerySet:
         if self.action == "list" and settings.CSV_REQUEST_PARAMETER in self.request.GET:
             for backend in (PatientDRYFilter, filters.DjangoFilterBackend):
                 queryset = backend().filter_queryset(self.request, queryset, self)
-            return queryset.filter(last_consultation__discharge_date__isnull=True)
+            return queryset.filter(
+                last_consultation__discharge_date__isnull=self.request.GET.get(
+                    "is_active"
```

Comment:
set the default value to false?

Reply:
Done, would've defaulted to false either way since dict values are strings and hence the comparison would evaluated to false.